debian/rules

```diff
@@ -29,12 +34,27 @@ export CARGO_HOME = $(CURDIR)/debian/cargo_home
 # Needed by the pam module loader
 export AUTHD_PAM_MODULES_PATH = /usr/lib/$(DEB_TARGET_GNU_TYPE)/security
 
+# Used not to hardcore the daemons paths
+export AUTHD_DAEMONS_PATH := /usr/libexec
+
 # Skip some tests that rely on external dependencies when building package:
 # they need external commands (e.g. `vhs`) that are not available in the build environment.
 export AUTHD_SKIP_EXTERNAL_DEPENDENT_TESTS=1
 
+# Defines the targets to be built as part of dh_auto_build
+export DH_GOLANG_BUILDPKG := $(AUTHD_GO_PACKAGE)/cmd/authd \
+	$(AUTHD_GO_PACKAGE)/pam \
+	$(NULL)
+
+# Run go generate on the targets
+export DH_GOLANG_GO_GENERATE := 1
```

Comment:
Run generate on which targets? The packages defined in `DH_GOLANG_BUILDPKG`?

Reply:
yep

Reply:
This should never happen. If we have something that needs to be generated, it should be done prior to the package build and, hopefully, tested as well.

Reply:
This always happened for us because `go generate on pam` implies *only* generating the pam library, in fact there we've an explicit check on existence of `"$PROJECT_ROOT"/vendor`.

So, we use generate as the tool for building the library, nothing else is really regenerated.

Then, as discussed before, what we're doing here (shipping generated code) is really something against debian policies, but let's ignore it.

Anyways, even previously we were doing `go generate` there, the only thing that it does is building the library

Reply:
Then this should be executed for `pam/` rather than also running on `cmd/authd/`, no?

Reply:
Technically yes, but given that we've nothing to generate in `cmd/authd` anyways, there's no point IMHO to increase the complexity on `debian/rules` and just make this go for both cases.

It will be just a dry-run call.